presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -157,6 +157,18 @@ public void stop()
         queryPurger.shutdownNow();
     }
 
+    /**
+     * Creates new queries on Presto.
+     * <p>
+     * Receives instruction requests that create new queries on Presto.
```

Comment:
Receives POST requests that tries to create new queries to be executed on Presto.